requirements.txt

```diff
@@ -23,4 +23,5 @@ biopython==1.76
 lz4==3.1.3
 scikit-bio==0.5.6
 ipywidgets==7.6.3
+ipython==8.12
```

Comment:
Do we need this?

Reply:
Unfortunately, yes. ipython > 8.12 dropped support for Python 3.8 and in some rare cases a newer version was pulled into the github workflow, causing errors.